project/Dependencies.scala

```diff
@@ -33,7 +33,8 @@ object Dependencies {
     val commonsCodec     = "1.15"
     val jodaTime         = "2.10.1"
     val useragent        = "1.21"
-    val uaParser         = "1.4.3"
+    val uaParser         = "1.5.3"
```

Comment:
Why bother bumping uap-java when we are explicitly pulling in snakeYaml, you might ask? 

https://github.com/ua-parser/uap-java/pull/71 

Hopefully this'll get merged and we can bump uap-java later, and remove the explicit snakeYaml dependency later, and someone can be saved finding out why [this](https://github.com/snowplow/enrich/pull/689/files#diff-659f5bb3bf18a2ab24ee6e50264b764a3cec9ef225f0a99cbbc3db2c8ea48efdL54) change was needed.